src/audio/eq_iir/llext/CMakeLists.txt

```diff
@@ -0,0 +1,53 @@
+# Copyright (c) 2024 Intel Corporation.
+# SPDX-License-Identifier: Apache-2.0
+
+cmake_path(SET SOF_BASE NORMALIZE ${CMAKE_CURRENT_SOURCE_DIR}/../../../..)
+
+set(MODULE "eq_iir")
+
+file(STRINGS ${CMAKE_CURRENT_LIST_DIR}/../${MODULE}.toml uuids REGEX "^[ \t]*uuid *=")
+
+foreach(line IN LISTS uuids)
+	# extract UUID value - drop the 'uuid = ' part of the assignment line
+	string(REGEX REPLACE "^[ \t]*uuid *= \"([0-9A-F\-]*)\"" "\\1" uuid ${line})
+	file(APPEND ${ZEPHYR_BINARY_DIR}/${MODULE}_llext/llext.uuid "${uuid}\n")
+endforeach()
```

Comment:
This is fairly complex regex, please don't copy/paste/diverge it in every module. Make this a macro or function in `zephyr/CMakeLists.txt`

Warning: CMake function can't "return" anything: they can only set "pseudo-return" variables. Find a very simple macro example in ef563ebfe3d5